store/store_test.go

```diff
@@ -4154,6 +4613,7 @@ func (s *storeTestSuite) TestBuy(c *C) {
 				if test.buyErrorCode == "" {
 					io.WriteString(w, test.buyResponse)
 				} else {
+					w.Header().Set("Content-Type", "application/json")
```

Comment:
is this a left of trying to change the main JSON request helper? do we have others?

Reply:
Yes, thanks for spotting. Removed for now. Yes there were some more tests that might need updating like this (just skimmed over them before, didn't investigate deeply).